### PR TITLE
add Ashish-Khanagwal

### LINKED ISSUE #1368: Ashish-Khanagwal
## Profile Screenshot

GitHub profile

### PR DESCRIPTION
**Pull request by Ashish-khanagwal**
resolve #1368

### CODE REVIEW

File: profiles/Ashish-Khanagwal.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/75534912/189076823-8aeefe43-1138-4837-8766-077da3817974.jpg",
+  "issueId": 1368,
+  "name": "Ashish Khanagwal",
+  "username": "Ashish-Khanagwal"
+}
```
Comment (Panquesito7) [inline profiles/Ashish-Khanagwal.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Comment (Ashish-khanagwal) [reply profiles/Ashish-Khanagwal.json:6]: Okay, I'll take care of it

Comment (Ashish-khanagwal) [reply profiles/Ashish-Khanagwal.json]: missing endline added, you can review now. If need further more changes then ping me.
Thank you

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**